### ISSUE TITLE
Multiple points for the same place??

### ISSUE THREAD

**Issue by aldofranco91** [ropensci/osmdata#169]
I want to download all the fire stations that are in Madrid. I am using the following code:

Bomberos = (opq(bbox = c(-3.888954,40.311977,-3.517916,40.643729),memsize = 4073741824,timeout = 400) %>% 
              add_osm_feature(key = 'amenity', value = "fire_station") %>%
              osmdata_sf())$osm_points

But my data frame has 603 observations because I am having multiple points for the same location.
Regards

**Comment by mpadge** [comment]
The following [`reprex`](https://github.com/tidyverse/reprex) illustrate what happens here:
``` r
library (osmdata)
#> Data (c) OpenStreetMap contributors, ODbL 1.0. http://www.openstreetmap.org/copyright
Bomberos <- opq(bbox = c(-3.888954,40.311977,-3.517916,40.643729)) %>%
    add_osm_feature(key = 'amenity', value = "fire_station") %>%
    osmdata_sf(quiet = FALSE)
#> Issuing query to Overpass API ...
#> Rate limit: 0
#> Query complete!
#> converting OSM data to sf format
table (Bomberos$osm_points$amenity, useNA = "always")
#> 
#> fire_station         <NA> 
#>            1          602
table (Bomberos$osm_polygons$amenity, useNA = "always")
#> 
#> fire_station         <NA> 
#>           64            2
index <- which (Bomberos$osm_polygons$amenity == "fire_station")
xy <- sf::st_coordinates (Bomberos$osm_polygons$geometry [index])
length (unique (rownames (xy)))
#> [1] 560
```

<sup>Created on 2019-02-28 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

There is only one point that is a "fire_station"; the rest are polygons. Those polygons themselves contain 560 unique points, and these are included in points. There are no repeated points, because `osmdata` never contains repeated points. Each point - indeed each object - in OSM has a unique identifier, and these are stored in `osmdata` as the rownames of all objects. That's why the final line is `unique (rownames (xy))`, not `length (!duplicated (xy))`. Points may share the same coordinates, yet still be different because they describe, or are part of, different things.

In short: if you want to extract the fire stations, you need to look at the polygons, as well as the one point, as well as the 2 multipolygons. (The latter contain the two polygons in `$osm_polygons` that do not have `amenity = "fire_station"`.) Hope that helps.
